packages/core/src/user-store/postgres/store.ts

```diff
@@ -92,57 +93,21 @@ export class PostgresUserStore implements UserStore {
         this.schema!.entities.map(async (model) => {
           const tableName = `${model.name}_${this.versionId}`;
           let tableBuilder = tx.schema.createTable(tableName);
-          model.fields.forEach((field) => {
-            switch (field.kind) {
-              case "SCALAR": {
-                tableBuilder = tableBuilder.addColumn(
-                  field.name,
-                  gqlScalarToSqlType[field.scalarTypeName],
-                  (col) => {
-                    if (field.notNull) col = col.notNull();
-                    return col;
-                  }
-                );
-                break;
-              }
-              case "ENUM": {
-                tableBuilder = tableBuilder.addColumn(
-                  field.name,
-                  "text",
-                  (col) => {
-                    if (field.notNull) col = col.notNull();
-                    col = col.check(
-                      sql`${sql.ref(field.name)} in (${sql.join(
-                        field.enumValues.map((v) => sql.lit(v))
-                      )})`
-                    );
-                    return col;
-                  }
-                );
-                break;
-              }
-              case "LIST": {
-                tableBuilder = tableBuilder.addColumn(
-                  field.name,
-                  "text",
-                  (col) => {
-                    if (field.notNull) col = col.notNull();
-                    return col;
-                  }
-                );
-                break;
-              }
-              case "RELATIONSHIP": {
-                tableBuilder = tableBuilder.addColumn(
-                  field.name,
-                  gqlScalarToSqlType[field.relatedEntityIdType.name],
-                  (col) => {
-                    if (field.notNull) col = col.notNull();
-                    return col;
-                  }
-                );
-                break;
-              }
+          Object.keys(model.columns).forEach((key) => {
+            if (!model.columns[key].list) {
```

Comment:
In general, I find it more readable to handle a truthy case first (even if it's less common/important). If you use `Object.entries`, this would read:

```ts
if (column.list) {
  // Handle scalar list columns.
  ...
} else {
  // Handle all other column types.
  ...
}
```

Which I find quite clear (comments optional 😅).